tools/validate_configs/test_configs/config-ssh.yaml

```diff
@@ -0,0 +1,99 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: hpc-slurm
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: hpc-small
```

Comment:
nit: deployment name and blueprint name are not in line with purpose of file.

Reply:
I changed the blueprint name, I left the deployment name because it aligns with hpc-slurm blueprint and so someone could use this blueprint without changes.